src/Monolog/Handler/SentryHandler.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Monolog\Handler;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Logger;
+use Sentry\Breadcrumb;
+use Sentry\Event;
+use Sentry\Severity;
+use Sentry\State\Hub;
+use Sentry\State\Scope;
+
+class SentryHandler extends AbstractProcessingHandler
+{
+    /**
+     * @var string the current application environment (staging|preprod|prod)
+     */
+    protected $environment;
+
+    /**
+     * @var string should represent the current version of the calling
+     *             software. Can be any string (git commit, version number)
+     */
+    protected $release;
+
+    /**
+     * @var Hub the hub object that sends the message to the server
+     */
+    protected $hub;
+
+    /**
+     * @var FormatterInterface The formatter to use for the logs generated via handleBatch()
+     */
+    protected $batchFormatter;
+
+    /**
+     * @param Hub $hub
+     * @param int $level The minimum logging level at which this handler will be triggered
+     * @param bool $bubble Whether the messages that are handled can bubble up the stack or not
+     */
+    public function __construct(Hub $hub, $level = Logger::DEBUG, bool $bubble = true)
```

Comment:
I think the first argument type should be `HubInterface`.

Reply:
Good catch!